README.md

```diff
@@ -34,10 +39,109 @@ Please help us by [Contributing](CONTRIBUTING.md) to the project.
 
 Here's the quickest way to get going with Rook.
 
-### Linux
+### Kubernetes
+
+This example shows how to build a simple, multi-tier web application on Kubernetes using persistent volumes enabled by Rook.
+
+#### Prerequisites
+
+This example requires a running Kubernetes cluster. To make sure you have a Kubernetes cluster that is ready for `rook`, you can [follow these quick instructions](demo/kubernetes/README.md).
+
+#### Deploy Rook
+
+With your Kubernetes cluster running, Rook can be setup and deployed by simply deploying the [rook-operator](demo/kubernetes/rook-operator.yaml) deployment manifest.
+You will find this manifest and all our example manifest files in the [demo/kubernetes](demo/kubernetes) folder.
+
+```
+cd demo/kubernetes
+kubectl create namespace rook
+kubectl create -f rook-operator.yaml
+```
+
+Use `kubectl` to list pods in the rook namespace. You should be able to see the following: 
+
+```
+$ kubectl -n rook get pod
+NAME                            READY     STATUS    RESTARTS   AGE
+mon0                            1/1       Running   0          1m
+mon1                            1/1       Running   0          1m
+mon2                            1/1       Running   0          1m
+osd-n1sm3                       1/1       Running   0          1m
+osd-pb0sh                       1/1       Running   0          1m
+osd-rth3q                       1/1       Running   0          1m
+rgw-1785797224-9xb4r            1/1       Running   0          1m
+rgw-1785797224-vbg8d            1/1       Running   0          1m
+rook-api-4184191414-l0wmw       1/1       Running   0          1m
+rook-operator-349747813-c3dmm   1/1       Running   0          1m
+```
```

Comment:
rgw should not be created at this level, no? If its a temporary I would make a note of it and reference the issue open around it.